### DISCUSSION TITLE
How can i install Mautic with postal smtp

### DISCUSSION THREAD

**Discussion by fav22** [postalserver/postal#1743] [Help with using Postal]
Hello , is it possible to install Mautic on server where there where i already have Postal smtp on it ?

If yes , how can i deal with it please ! Thank you all

**Comment by willpower232** [comment]
Ideally you would keep it running separately to avoid any risk of running out of CPU or memory but as long as your server has sufficient resources to run everything then there probably isn't a problem as it doesn't look like there are any port conflicts.

**Reply by fav22** [reply]
Hello , sorry for the late reply (due of timezone) , i think i have sufficient ressources in my server to run everything !
How do you think about my machine :
 CPU
4 vCPU Cores

RAM
8 GB RAM

STORAGE
50 GB NVMe

or 200 GB SSD

DATA TRANSFER
32 TB Traffic

Unlimited Incoming

If it's ok , can you suggest me how to install mautic correctly , i always have an error when creating database and SSL let's enscrypt , the same when i install let's encrypt on postal

**Reply by fav22** [reply]
i can always upgrade my vps if it necessary !

**Reply by willpower232** [reply]
What error messages do you have?

**Reply by fav22** [reply]
```
Obtaining a new certificate
Performing the following challenges:
http-01 challenge for postal.mydomain.org
Using the webroot path /opt/postal/app/public for all unmatched domains.
Waiting for verification…
Challenge failed for domain postal.mydomain.org
http-01 challenge for postal.mydomain.org
Cleaning up challenges
Some challenges have failed.
```


`– The following errors were reported by the server:

Domain: postal.mydomain.org
Type: unauthorized
Detail: Invalid response from
https://postal.mydomain.org/.well-known/acme-challenge/wTVdxm0GyfEiUEr XVjsKdTLrVg1esDxGa5NR2UMRs0Y
[xxxxxxxx]: 502

To fix these errors, please make sure that your domain name was
entered correctly and the DNS A/AAAA record(s) for that domain
contain(s) the right IP address.
– Your account credentials have been saved in your Certbot
configuration directory at /etc/letsencrypt. You should make a
secure backup of this folder now. This configuration directory will
also contain certificates and private keys obtained by Certbot so
making regular backups of this folder is ideal.`

**Reply by willpower232** [reply]
> Challenge failed for domain postal.mydomain.org

this implies that the DNS record for postal.mydomain.org is not working
